client/src/components/History/Export/HistoryExport.vue

```diff
@@ -24,6 +29,8 @@ const props = defineProps({
     },
 });
 
+const POLLING_DELAY = 10000;
```

Comment:
```suggestion
const POLLING_DELAY = 3000;
```

that lines it up with other polling delays